src/pages/newinsertion/index.tsx

```diff
@@ -0,0 +1,7 @@
+import Insertion from '../../components/Insertion';
+
+const NewInsertion: React.FC = () => {
+  return <Insertion />;
```

Comment:
perche non mettere direttamente il componente qui? non ti serve aggiungere questo step in piu dove esporti una pagina che riesporta il componente, sposta direttamente il componente in questo file

Reply:
Secondo me era ordinato avere delle cartelle sul progetto con solo il nome della pagina, ma non ha nessun senso. Quindi direi che inserisco direttamente dentro il componente senza passare dalla pagina e tolgo anche le altre

Reply:
no ma e' ordinato, ci sta, pero appunto che hai una cartella con le rotte, mettici dentro la rotta, e non forzarti a creare un componente separato quando gia ce l'hai

Reply:
Devo ancora capire come strutturare il tutto bene